### PR TITLE
Add basic parameters to more scripts

### PR DESCRIPTION
**Pull request by hannob**
Okay, some more additions of parameters. If these look okay to you I'll continue with the rest.

I also tried to somewhat unify the variable names to what's used in test_conversations.py, this could be done for all scripts (e.g. hostname/host, conversation_name/c_name).

### CODE REVIEW

File: scripts/test-bleichenbacher-workaround.py [hunk 1]
```diff
@@ -17,11 +18,51 @@
...
+
+def help_msg():
+    print("Usage: <script-name> [-h hostname] [-p port] [[probe-name] ...]")
+    print(" -h hostname    name of the host to run the test against")
+    print("                localhost by default")
+    print(" -p port        port number to use for connection, 4433 by default")
+    print(" probe-name     if present, will run only the probes with given")
+    print("                names and not all of them, e.g \"sanity\"")
+    print(" -e probe-name  exclude the probe from the list of the ones run")
+    print("                may be specified multiple times")
```
Comment (tomato42) [inline scripts/test-bleichenbacher-workaround.py]: `-n` undocumented

**Comment by tomato42** [comment]
sorry for the delay in reviewing

**Comment by hannob** [comment]
Sorry, fixed. Are you okay with it like that or do you want me to rewrite git history?

File: scripts/test-bleichenbacher-workaround.py [hunk 2]
```diff
@@ -17,11 +18,49 @@
...
+
 def main():
     """Check if server is not vulnerable to Bleichenbacher attack"""
+    host = "localhost"
+    port = 4433
+    num_limit = None
+    run_exclude = set()
+
+    argv = sys.argv[1:]
+    opts, args = getopt.getopt(argv, "h:p:e:n:", ["help"])
```
Comment (tomato42) [inline scripts/test-bleichenbacher-workaround.py]: `-n` still present

File: scripts/test-fuzzed-MAC.py [hunk 3]
```diff
@@ -17,8 +18,46 @@
...
+
 def main():
     """check if incorrect MAC hash is rejected by server"""
+    host = "localhost"
+    port = 4433
+    num_limit = None
+    run_exclude = set()
+
+    argv = sys.argv[1:]
+    opts, args = getopt.getopt(argv, "h:p:e:n:", ["help"])
```
Comment (tomato42) [inline scripts/test-fuzzed-MAC.py]: `-n` present

File: scripts/test-fuzzed-padding.py [hunk 4]
```diff
@@ -17,11 +18,47 @@
...
     """check if incorrect padding is rejected by server"""
-    conversations = {}
-    hostname = "localhost"
+    host = "localhost"
     port = 4433
+    num_limit = None
+    run_exclude = set()
+
+    argv = sys.argv[1:]
+    opts, args = getopt.getopt(argv, "h:p:e:n:", ["help"])
```
Comment (tomato42) [inline scripts/test-fuzzed-padding.py]: `-n` present

**Comment by tomato42** [comment]
> Are you okay with it like that or do you want me to rewrite git history?

in general I prefer history rewrites - makes bisecting much more sane later -  in this case there are no changes to functionality at any point so I could make an exception

**Comment by tomato42** [comment]
@hannob why did you close this PR? it seems like you overwrote your master...

**Comment by hannob** [comment]
Sorry, I'm in the process of learning how to better use git, github and rewrite git history...

Now pull request should be updated with your requested changes and a clean history of three commits.

**Comment by tomato42** [comment]
perfect! thanks for bearing with my nitpicking

**Comment by tomato42** [comment]
BTW: it's usually best to keep your own `master` clean (and equal to upstream `master`) and do development in a feature branch - this makes updating your own repo much easier/cleaner when upstream modifies master

**Pull request merged**